README.md

```diff
@@ -49,6 +49,32 @@ To deploy a Opennebula instance locally run:
 
 where "boxname" can be debian or centos
 
+## Docker
+
+To deploy a Opennebula instance locally in a docker container run these commandos.
+
+Build base image with puppet in it (Depending on centos:latest):
+
+    docker build --rm -f docker/Dockerfile-Centos6 -t epost-dev/puppet .
+
+Build a image with the one puppet module in it:
+
+    docker build --rm -f docker/Dockerfile -t epost-dev/one .
+
```

Comment:
Please provide a better description.
The first docker build command is needed to build an image with puppet installed.
The second command makes use of the image built in the first command.
This is not very obvious.